make/modules/java.base/Java.gmk

```diff
@@ -26,6 +26,8 @@
 DOCLINT += -Xdoclint:all/protected \
     '-Xdoclint/package:java.*,javax.*'
 JAVAC_FLAGS += -XDstringConcat=inline
```

Comment:
I think it makes more sense to just have a single `JAVAC_FLAGS` assignment line, not one per flag.